osu.Game/Overlays/Settings/SettingsFooter.cs

```diff
@@ -17,7 +18,13 @@ namespace osu.Game.Overlays.Settings
 {
     public class SettingsFooter : FillFlowContainer
     {
-        [BackgroundDependencyLoader]
+        [Resolved]
+        private OsuGameBase game { get; set; }
+
+        [Resolved(CanBeNull = true)]
+        private ChangelogOverlay changelog { get; set; }
+
+        [BackgroundDependencyLoader(true)]
```

Comment:
Remove `(true)` unless you explicitly want the arguments to all null values.

Reply:
Whoops forgot to remove that one in the previous commit.